packages/@okta/vuepress-site/docs/guides/dpop/main/oktaresourceserver/requestresource.md

```diff
@@ -0,0 +1,47 @@
+## Make a request to an Okta resource
+
+<ApiLifecycle access="ea" />
+
+Access to an Okta resource requires more steps.
+
+1. Hash and base64url-encode the DPoP-bound access token for use as the `ath` value.
+1. Use the [Create the JSON Web Token](#create-the-json-web-token) section to create a DPoP proof JWT with the following claims:
```

Comment:
Should number be a 2 here.  Curious if markdown has autonumbering. I've seen what looks like text 1,2,3... above.

Reply:
Yes, it depends if it has a break or not. This doesn't have a break, so this is fine.